AjaxControlToolkit/BarChart/BarChart.cs

```diff
@@ -105,6 +105,11 @@ public class BarChart : ChartBase {
                     throw new Exception("Name is missing the BarChartSeries. Please provide a name in the BarChartSeries.");
             }
         }
+
+        protected override void DescribeComponent(ScriptComponentDescriptor descriptor) {
+            base.DescribeComponent(descriptor);
+            descriptor.AddElementProperty("container", ContainerID);
```

Comment:
No, this is not an ID.
Please take note of the `Element` part in the `AddElementProperty` method - it resolves a DOM element on the client side automatically with the passed `ContainerID`.